src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -98,6 +98,22 @@ public static List<MCItem> loadBlocksAndItemsAndTags(Workspace workspace) {
 		return elements;
 	}
 
+	/**
+	 * Loads all mod elements and all Minecraft blocks, including elements
+	 * that are wildcard elements to subtypes (wood -&gt; oak wood, birch wood, ...)
+	 *
+	 * @return All Blocks from both Minecraft and custom elements with or without metadata
+	 */
+	public static List<MCItem> loadBlocksAndTags(Workspace workspace) {
```

Comment:
This method is exact copy of loadBlocks. What is the purpose of it then? If you meant to load wildcard elements, then you should remove hasNoSubtypes

Reply:
You probably wanted to load wildcard elements here, right? Not sure your replacement to loadBlocks only was the solution here.

Now one can not specify "any block" in the selectors that support tags, while this would be a reasonable thing to expect to work I think.